app/controllers/site_controller.rb

```diff
@@ -110,6 +110,7 @@ def edit
   end
 
   def copyright
+    @title = t ".title"
```

Comment:
Should this use the page locale rather than the user's locale? In other words move it after `@locale` is set and add `, :locale => @locale` to it?

Reply:
Yeah, this could be decided either way. I'm happy with having the title in the same language as the rest of the chrome, so I'm happy to merge as-is.